### ISSUE TITLE
ImportError: cannot import name '_expand_mask' from 'transformers.models.bart.modeling_bart'

### ISSUE THREAD

**Issue by MorenoLaQuatra** [ccdv-ai/convert_checkpoint_to_lsg#8]
First and foremost, thank you for your efforts put into the project. Your contributions are valuable to the whole research community.

However, I recently encountered an issue while attempting to import the LSGConverter with the latest version of the transformers library (4.35.2). The error I encountered is as follows:

```python
from lsg_converter import LSGConverter

> from transformers.models.bart.modeling_bart import _expand_mask
ImportError: cannot import name '_expand_mask' from 'transformers.models.bart.modeling_bart' (/MY_INTERNAL_PATH_TO/transformers/models/bart/modeling_bart.py)
```

Interestingly, this issue does not arise when using version 4.29.1 of the transformers library.

**Comment by MorenoLaQuatra** [comment]
I also try to add another question. While using the LSG version of the model, I noticed an error with `position_embeddings`

```
RuntimeError: Error(s) in loading state_dict for LSGBertModel:
        size mismatch for bert.embeddings.position_ids: copying a param with shape torch.Size([1, 512]) from checkpoint, the shape in current model is torch.Size([1, 2048]).
        You may consider adding `ignore_mismatched_sizes=True` in the model `from_pretrained` method.
```

I'm using a `bert-base-cased` model extended to 2048 tokens. Adding `ignore_mismatched_sizes=True` the problem is solved, however, I don't know if it is the correct approach.

**Comment by ccdv-ai** [comment]
Hi @MorenoLaQuatra 
The `_expand_mask` problem should be fixed now with `lsg-converter 0.1.8`, please update your package.
I cannot reproduce your second problem. Can you provide some code?

**Comment by MorenoLaQuatra** [comment]
Updating both the `lsg-converter` and `transformers` version solved both, maybe it was due to a mismatch in versions. Thank you!
